src/calculateRentalCost.js

```diff
@@ -4,7 +4,22 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dailyPrice = 40;
+  const totalCost = dailyPrice * days;
+  const longTerm = 7;
+  const shortTerm = 3;
+  const longTermDiscount = 50;
+  const shortTermDiscount = 20;
+
+  if (days >= longTerm) {
+    return days * dailyPrice - longTermDiscount;
+  }
+
+  if (days >= shortTerm) {
+    return days * dailyPrice - shortTermDiscount;
```

Comment:
This condition applies the short-term discount if the number of days is greater than or equal to the short-term threshold. However, if the long-term condition is also met, this block will never be executed. Consider restructuring the logic to ensure that the correct discount is applied based on the number of days.